README.md

```diff
@@ -1,5 +1,12 @@
 # Tutorial for Husky
 
+---
+
+**Note:**
+These tutorials have been superceded. Find the latest Husky tutorials at [docs.clearpathrobotics.com](https://docs.clearpathrobotics.com/docs/robots/husky/husky).
```

Comment:
Is this supposed to be `https://docs.clearpathrobotics.com/docs/robots/husky/tutorials_husky`?